public/editor-client/src/index.ts

```diff
@@ -13,6 +14,10 @@ const api = {
     addMedia,
     mediaResizeUrl: config.actions.mediaResizeUrl
   },
+  customFile: {
+    addFile,
+    fileUrl: config.actions.fileUrl
```

Comment:
Use config.api.customFile.customFileUrl() as you will delete the keys from actions.